views/pulsar/v2/helpers/form.html.twig

```diff
@@ -948,11 +948,19 @@ controls.class   | string | A space separated list of class names
             {% set input_group_classes = input_group_classes ~ ' is-removable' %}
         {% endif %}
 
+        {% if (options.parent.prepend_type is defined and options.parent.prepend_type == 'button') %}
```

Comment:
It'll just not print out the classes, or the button, I presume (as it uses the same if statement as on line 965)